### ISSUE TITLE
setup-watcom broken via recent commits

### ISSUE THREAD

**Issue by bradh352** [open-watcom/setup-watcom#72]
Looks like the recent changes (updating to NodeJS v20, etc) have broken setup-watcom.  See an example here:
https://github.com/c-ares/c-ares/actions/runs/7733830705/job/21117277111

```
Run open-watcom/setup-watcom@v0
  with:
    version: [2](https://github.com/c-ares/c-ares/actions/runs/7733830705/job/21117277111#step:2:2).0
Initializing action.
 Error: Input does not meet YAML 1.2 "Core Schema" specification: environment
  Support boolean input list: `true | True | TRUE | false | False | FALSE`
```

**Comment by jmalak** [comment]
No, it is issue with OW build on GitHub, see OW issue [1222](https://github.com/open-watcom/open-watcom-v2/issues/1222) .

**Comment by bradh352** [comment]
Really?  Doesn't look like to me its even getting that far, looks like a parse failure of `action.yml` itself due to the `default: true` being removed from `environment`.  I'd guess if I added `environment: true` to the c-ares github action it would continue on. 
https://github.com/open-watcom/setup-watcom/compare/8a33373..01d689c

**Comment by jmalak** [comment]
Sorry, I didn't study much this issue.
Anyway now setup-watcom cannot work due to referenced issue.

**Comment by madebr** [comment]
https://github.com/open-watcom/setup-watcom/pull/73 fixes this, I think.
The default value of `environment` should not have been removed.

It fails for 2.0 because of https://github.com/open-watcom/open-watcom-v2/issues/1222
